src/stories/CardMeta.stories.js

```diff
@@ -291,3 +293,40 @@ export function FtvaLinkedCategoryAndTitle() {
   `,
   }
 }
+
+export function FtvaCustomTitleAndDesription() {
```

Comment:
@jendiamond RichText styling is not taking effect in this story's title field. The inspector is still showing that <rich-text> tag.